src/c++/perf_analyzer/genai-perf/genai_perf/llm_inputs/llm_inputs.py

```diff
@@ -154,12 +154,12 @@ def create_llm_inputs(
                 dataset
             )
         elif input_type == PromptSource.SYNTHETIC:
+            random.seed(random_seed)
```

Comment:
I think we always pass random seed of 0 as a default. Is this expected?

Reply:
Yes, that is the default into the function. When we use it in real code, it comes from the args (which also defaults to 0, which is fine):
https://github.com/triton-inference-server/client/blob/main/src/c%2B%2B/perf_analyzer/genai-perf/genai_perf/main.py#L76

Reply:
yes, but if we set random seed to 0 every time, I think we will have same prompt every time. Is that also expected?

Reply:
Yes. That is by design. We want this to be reproducible so people can do apples to apples comparisons. Given the same requested input token mean and stddev, we should produce the same prompts.

If people want to run over and over with different data, they can do so by changing the seed.

Reply:
To be clear, each prompt for a given run will all be different. Those set of prompts will be the same if a user runs again without changing the seed.